submissions/xT0PERx/Popup-style/style.css

```diff
@@ -0,0 +1,94 @@
+/*                                         Верхнее меню с 5 блоками  */
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    font-size: 1em;
+    text-align: center;
+    padding: 5px;
+}
+
+.container div {
+    width: 80px;
+    height: 80px;
+    margin: 5px;
+    cursor: pointer;
+}
+
+img {
+    width: 40px;
+    height: 40px;
+}
+
+label img:hover {
```

Comment:
Avoid jumping content like on the [video](https://share.vidyard.com/watch/ENmC1WuPCdzReXApqha2QX?)
Add a transparent border to element(to reserve place to avoid jumping) and then just change border-color on `:hover`